components/src/main/java/uk/gov/hmrc/components/organism/information/InformationMessageCardView.kt

```diff
@@ -80,14 +80,24 @@ class InformationMessageCardView @JvmOverloads constructor(
         binding.warningView.setIcon(resId)
     }
 
-    fun setTitle(title: CharSequence?) {
+    fun setContentTitle(title: CharSequence?) {
         binding.contentTitle.text = title
-        binding.contentTitle.visibility = if (title != null) View.VISIBLE else View.GONE
+        binding.contentTitle.visibility = if (!title.isNullOrEmpty()) View.VISIBLE else View.GONE
+        if (!title.isNullOrEmpty() &&
+            binding.contentBody.text.isNullOrEmpty() &&
+            binding.buttonsContainer.childCount == 0) {
+            binding.viewBottomPadding.visibility = View.VISIBLE
+        }
+        updateContentContainerVisibility()
     }
 
-    fun setBody(body: CharSequence?) {
+    fun setContentBody(body: CharSequence?) {
         binding.contentBody.text = body
-        binding.contentBody.visibility = if (body != null) View.VISIBLE else View.GONE
+        binding.contentBody.visibility = if (!body.isNullOrEmpty()) View.VISIBLE else View.GONE
```

Comment:
Bit of mind bend with "if not is null or is not empty"

Might it read better like?

```kotlin
binding.contentBody.visibility = if (body.isNullOrEmpty()) View.GONE else View.VISIBLE
```

Reply:
But not precious about it if you disagree

Reply:
This change would definitely have made sense! However, just deleted it now as the requirements have changed. :)